src/lib/mina_base/transaction_logic.ml

```diff
@@ -1633,8 +1633,8 @@ module Make (L : Ledger_intf) : S with type ledger := L.t = struct
                    (Control.tag p.authorization))
               ~has_proof:(Control.Tag.equal (Control.tag p.authorization) Proof)
               ~is_new:(match loc with `Existing _ -> false | `New -> true)
-              ~current_global_slot:global_state.protocol_state.curr_global_slot
-              p.data a
+              ~current_global_slot:
```

Comment:
Can you rename this argument, for completeness?

Reply:
Done.